scripts/arangodb/start-arango-db.sh

```diff
@@ -10,49 +10,55 @@ set -e
 # ARANGO_DB_ROOT_PASSWORD="rootPASSWORD";
 # ARANGO_DB_USER="devtester";
 # ARANGO_DB_USER_PASSWORD="confidential"
-# ARANGO_DB_NAME="testdb";
+# ARANGO_DB_NAME="coscradtestdb";
 
-if [ $1 == "test_arango" ];
-then
-  echo "Setting up test data for ArangoDB";
-  PROJECT_ROOT_PATH="$PWD";
-  echo "PROJECT_ROOT_PATH: $PROJECT_ROOT_PATH"
-  ARANGO_DOCKER_LOCAL_DIR="../arango-volume-share";
-  ARANGO_DOCKER_VOLUME_DESTINATION="/home/arango-volume-share";
-
-  # Create arango volume share location
-  if [ -d "$ARANGO_DOCKER_LOCAL_DIR" ]; then
-    echo "Removing $ARANGO_DOCKER_LOCAL_DIR"
-    rm -rf "$ARANGO_DOCKER_LOCAL_DIR"
-  fi
+# Load path and script vars
+. ./scripts/arangodb/arango_setup_vars.sh
 
-  mkdir $ARANGO_DOCKER_LOCAL_DIR
-  echo "Created local Docker share directory $ARANGO_DOCKER_LOCAL_DIR"
+# Register setup mode
+if [ $1 ]; then
+  SETUP_MODE=$1
+  export SETUP_MODE;
+else
+  echo "No setup mode input argument supplied.  Accepted: dev | staging | prod";
```

Comment:
Nice approach. I wasn't thinking we'd use this script for production. I figured we'd have a permanent Arango instance with production and close-to-production instances that we never ever want to take down. But given your backup and restore utilities and mounting to persistent volumes, we could make the production Arango instance itself disposable. Something to think about!